.github/dependabot.yaml

```
@@ -0,0 +1,15 @@
+version: 2
+updates:
+
+  # Maintain dependencies for the root package
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

By default, the day is on Monday, so if we are going to check the dependencies on Friday, we should move it to Friday. Also, the time, by default its a random time, but we can set an hour.

https://docs.github.com/es/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#scheduleday

```suggestion
      interval: "weekly"
      day: "friday"
      time: "08:00"
```

---

Add the `timezone: "Europe/Madrid"` property too, so it is always based in our timezone instead of defaulting to UTC.